src/app/jscaip/AI/BoardValue.ts

```diff
@@ -0,0 +1,54 @@
+import { Utils } from 'src/app/utils/utils';
+import { Player } from '../Player';
+
+export class BoardValue {
+
+    public static VICTORIES: number[] = [Number.MIN_SAFE_INTEGER, Number.MAX_SAFE_INTEGER];
+
+    public static isVictory(score: number): boolean {
+        return score === Number.MAX_SAFE_INTEGER || score === Number.MIN_SAFE_INTEGER;
+    }
+
+    public static isPreVictory(score: number): boolean {
+        return score === Number.MAX_SAFE_INTEGER - 1 || score === Number.MIN_SAFE_INTEGER + 1;
+    }
+
+    /**
+     * Returns the board value link to those player's scores
```

Comment:
linked* players'*
ou simplement: return the board value corresponding to the players' scores